packages/snaps-cli/src/webpack/config.ts

```diff
@@ -0,0 +1,320 @@
+     */
+    module: {
+      rules: [
+        {
+          test: /\.[tj]sx?$/u,
+          exclude: /node_modules/u,
+          use: await getDefaultLoader(config),
+        },
+
+        config.experimental.wasm && {
+          test: /\.wasm$/u,
+          use: {
+            loader: resolve(__dirname, 'loaders', 'wasm'),
+          },
+        },
+      ],
+    },
+
+    /**
+     * The resolve configuration. This tells Webpack how to resolve imports.
+     * We set it to resolve `.js` and `.ts` files.
+     *
+     * @see https://webpack.js.org/configuration/resolve/
+     */
+    resolve: {
+      /**
+       * The extensions to resolve. We set it to resolve `.js` and `.ts`
+       * files.
+       */
+      extensions: ['.js', '.ts'],
+
+      /**
+       * The fallback options. This tells Webpack how to handle imports that
+       * aren't resolved. By default, we set Node.js built-ins to `false`, so
+       * that they are ignored.
+       */
+      fallback: Object.fromEntries(builtinModules.map((name) => [name, false])),
+
+      /**
+       * The plugins to use. We use the {@link SnapsBuiltInResolver} to show
+       * warnings about using Node.js built-ins, when no fallback is specified.
+       */
+      plugins: [builtInResolver],
+    },
+
+    /**
+     * The plugins to use.
+     *
+     * @see https://webpack.js.org/configuration/plugins/
+     */
+    plugins: [
+      /**
+       * The `SnapsWebpackPlugin` is a Webpack plugin that checks and updates
+       * the manifest file, and evaluates the bundle in SES. While not strictly
+       * required, it's highly recommended to use this plugin.
+       */
+      new SnapsWebpackPlugin({
+        manifestPath: config.manifest.path,
+        writeManifest: config.manifest.update,
+        eval: options.evaluate,
+      }),
+
+      /**
+       * The `SnapsStatsPlugin` is a Webpack plugin that handles the stats
+       * output. It's used to show the stats in the terminal, in a format that
+       * is easy to read.
+       */
+      new SnapsStatsPlugin(config.plugins.stats, options.spinner),
+
+      /**
+       * The `EnvironmentPlugin` is a Webpack plugin that adds environment
+       * variables to the bundle. We use it to add the `NODE_ENV` and `DEBUG`
+       * environment variables.
+       */
+      new EnvironmentPlugin(config.environment),
+
+      /**
+       * The `ProgressPlugin` is a Webpack plugin that logs the progress of
+       * the build. We set it to log the progress to the spinner.
+       */
+      new ProgressPlugin({
+        handler: getProgressHandler(options.spinner, spinnerText),
+      }),
+
+      /**
+       * The `WatchPlugin` is a Webpack plugin that adds extra files to watch
+       * for changes. This is useful for rebuilding the bundle when the
+       * manifest file changes.
+       */
+      options.watch &&
+        new SnapsWatchPlugin(
+          {
+            files: [config.manifest.path],
```

Comment:
It seems like we don't watch `snap.config.js` anymore?

Reply:
The Webpack compiler isn't recreated for every change, so there doesn't seem to be any benefit in watching the config. I'm not sure if there was before?

Reply:
It might not have made much difference before either 🤔